.github/workflows/build.yml

```diff
@@ -48,6 +43,12 @@ jobs:
             - name: Lock Symfony version
               run: VERSION=${{ matrix.symfony-version }} .github/workflows/lock-symfony-version.sh
 
+            - name: Remove Symfony proxy-manager-bridge
+              run: |
+                  composer config --no-plugins allow-plugins.dealerdirect/phpcodesniffer-composer-installer true
```

Comment:
the allowed plugins should be configured in your composer.json directly to avoid bothering contributors each time.